content/en/docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -53,6 +53,33 @@ Minikube can be installed locally, and runs a simple, single-node Kubernetes clu
 
 You can get basic information about your cluster with the commands `kubectl cluster-info` and `kubectl get nodes`. However, to get a good idea of what's really going on, you need to deploy an application to your cluster. This is covered in the next section.
 
+#### microk8s
+
+On Linux, *microk8s* might be a good alternative to Minikube for a
+local install of Kubernetes. Microk8s keeps up with the current
```

Comment:
This is tricky but it seems from your heading that "microk8s" shouldn't be capitalized. Can you rewrite so it's not the first word of any sentence, to avoid this awkward situation?